lib/ios-deploy.js

```diff
@@ -45,11 +45,11 @@ class IOSDeploy {
     const installationService = await services.startInstallationProxyService(this.udid);
     const appInstalledNotification = new B((resolve) => {
       notificationService.observeNotification(APPLICATION_INSTALLED_NOTIFICATION, { notification: resolve });
-    }).timeout(APPLICATION_NOTIFICATION_TIMEOUT);
+    });
     try {
       await installationService.installApplication(bundlePathOnPhone, { PackageType: 'Developer'});
       try {
-        await appInstalledNotification;
+        await appInstalledNotification.timeout(APPLICATION_NOTIFICATION_TIMEOUT);
       } catch (e) {
         log.warn(`Couldn't get the application installed notification with ${APPLICATION_NOTIFICATION_TIMEOUT}ms but we will continue`);
```

Comment:
how about logging the actual error here?

Reply:
Sure, I can add that